WordPress/Classes/Services/SiteCreationService.swift

```diff
@@ -147,10 +138,24 @@ open class SiteCreationService: LocalCoreDataService {
 
         let createBlogFailureBlock: SiteCreationFailureBlock = { error in
             WPAppAnalytics.track(.createSiteCreationFailed)
+            DDLogError("Error while creating site: \(String(describing: error))")
             failure(error)
         }
 
         let validateBlogSuccessBlock = {
+
```

Comment:
> Can you explain?

It made more sense logically to test for the default account just before it was used, instead of some seemingly random place.

> I'm a bit unsure if this would have implications for the old flow.

The `SignupService` has it's own `validateBlogSuccessBlock`, and `SignupService` doesn't call `createSite` directly (it calls the methods it needs directly) so there is no impact.

Reply:
Got it—thanks!

PROCEED